src/menhir.ml

```diff
@@ -0,0 +1,32 @@
+open Import
+
+let gen_rules sctx ~dir ~scope (t : Jbuild.Menhir.t) =
+  let targets n = List.map ~f:(Path.relative dir) [n ^ ".ml"; n ^ ".mli"] in
+  let flags =
+    List.map ~f:(Super_context.expand_vars sctx ~scope ~dir) t.flags in
```

Comment:
For uniformity with the rest of Jbuilder, we should change that so that flags use an unexpanded ordered set language

Reply:
OK, I was meaning to do that anyway. Just didn't want to introduce new behavior in a refactor.